src/analyser/index.tsx

```diff
@@ -25,6 +29,26 @@ export class Analyser {
     if (source) { source.disconnect(); }
   }
 
+  getBucketedByteFrequencyData = (maxNumBuckets: number): Array<number> => {
```

Comment:
Move logic for bucketing frequency data into `Analyser` (other visualisations can use this)